processes.py

```diff
@@ -318,3 +319,41 @@ def process_to_dict(process):
     models.FEATURE_TYPE_CODE_CHANGE_ID: PSA_ONLY_PROCESS,
     models.FEATURE_TYPE_DEPRECATION_ID: DEPRECATION_PROCESS,
     }
+
+
+# These functions return a true value when the checkmark should be shown.
+# If they return a string, and it starts with "http:" or "https:", it will
+# be used as a link URL.
+PROGRESS_DETECTORS = {
+    'Initial public proposal':
+    lambda f: f.initial_public_proposal_url,
+
+    'Explainer':
+    lambda f: f.explainer_links and f.explainer_links[0],
+
+    'Samples':
+    lambda f: f.sample_links and f.sample_links[0],
+
+    'TAG review request':
+    lambda f: f.tag_review,
+
+    'Request signals':
+    lambda f: bool(
+        f.ff_views != models.NO_PUBLIC_SIGNALS or
+        f.safari_views != models.NO_PUBLIC_SIGNALS or
+        f.ie_views != models.NO_PUBLIC_SIGNALS),
+
+    'Estimated target milestone':
+    lambda f: bool(f.shipped_milestone),
+
+    'Code in Chromium':
```

Comment:
Is there also a "Code in repo"

Reply:
Yes, but I believe that refers to a GitHub repo or something for the WICG.  I don't think we have a field for the user to specify that yet.